src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -28,6 +28,43 @@
     </Content>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Remove="CFamily\CLangAnalyzerRequest.json" />
```

Comment:
Was there are reason for the request file not to be in the `IntegrationTests` folder with the the other resource files?

Reply:
Hmm that's a mistake, the file was renamed to CLangAnalyzerRequestTemplate and moved to the correct folder, must be a left-over.